packages/react/src/ui/Connect/components/Connector/Connector.tsx

```diff
@@ -25,15 +23,14 @@ export function Connector() {
   } = connector.metadata;
 
   // Ping extension: if it's installed, it will trigger connector
-  useQuery({
-    queryKey: ['CONNECTOR_PING', connector.name, connector.installed],
-    queryFn: async () => {
-      const isInstall = await connector.ping();
-      if (isInstall) setRoute(Routes.CONNECTING);
-      return isInstall;
-    },
-    staleTime: Number.POSITIVE_INFINITY,
-  });
+  useEffect(() => {
+    const ping = async () => {
```

Comment:
I'd rather have this with react query + refetch interval, for the Wallets that can be detected after installation without reloading the page (e.g. our Wallet)